pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -69,9 +69,12 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 	kapi.Resource("podtemplates"):                               kapi.Kind("PodTemplate"),
 	kapi.Resource("replicationcontrollers"):                     kapi.Kind("ReplicationController"),
 	batch.Resource("jobs"):                                      batch.Kind("Job"),
+	batch.Resource("jobtemplates"):                              batch.Kind("JobTemplate"),
```

Comment:
I expect these to be paired with a change to `getPodSpec`.  I really should fix that factorization.